draft-ietf-anima-brski-prm.md

```diff
@@ -336,84 +320,84 @@ Solution examples based on existing technology are provided with the focus on ex
+|        |     .  |            |        |           |  .
+| Pledge | BRSKI- | Registrar- | BRSKI- | Domain    |  .
+|        |  PRM   | Agent      |  EST   | Registrar |  .
+|        |<------>|            |<------>| (PKI RA)  |  .
+|        |     .  |     LDevID |        |           |  .
+|        |     .  +------------+        +-----+-----+  .
+| IDevID |     .                              |        .
+|        |     .           +------------------+-----+  .
++--------+     .           | Key Infrastructure     |  .
+               .           | (e.g., PKI Certificate |  .
+               .           |        Authority)      |  .
+               .           +------------------------+  .
+               .........................................
+                         "Domain" Components
 ~~~~
-{: #uc2figure title='BRSKI-PRM Architecture overview using registrar-agent' artwork-align="left"}
+{: #uc2figure title='BRSKI-PRM architecture overview using registrar-agent' artwork-align="left"}
 
 The following list describes the components in a (customer) site domain:
 
 * Pledge: The pledge is expected to respond with the necessary data objects for bootstrapping to the registrar-agent.
   The protocol used between the pledge and the registrar-agent is assumed to be HTTP in the context of this document.
-  Other protocols may be used like CoAP, Bluetooth, or NFC, but are out of scope of this document.
-  A pledge acting as a server during bootstrapping leads to some differences to BRSKI:
+  Other protocols such as CoAP, NFC, or Bluetooth may be used, but are out of scope of this document.
+  A pledge acting as a server during bootstrapping leads to some differences for BRSKI:
 
   * Discovery of the pledge by the registrar-agent must be possible.
 
   * As the registrar-agent must be able to request data objects for bootstrapping of the pledge, the pledge must offer corresponding endpoints.
 
-  * The registrar-agent MAY provide additional data to the pledge in the context of the voucher triggering request, that the pledge includes into the PVR.
+  * The registrar-agent MAY provide additional data to the pledge in the context of the voucher-request trigger, which the pledge includes into the PVR.
     This allows the registrar to identify, with which registrar-agent the pledge was in contact.
 
   * Order of exchanges in the call flow may be different as the registrar-agent collects both, PVR and PER, at once and provides them to the registrar.
-    This approach is used in order to allow for bulk bootstrapping of several devices in a single pass through a new site by the commissioning personnel.
+    This enables bulk bootstrapping of several devices.
 
   * The data objects utilized for the data exchange between the pledge and the registrar are self-contained authenticated objects (signature-wrapped objects).
 
-
 * Registrar-agent: provides a communication path to exchange data objects between the pledge and the domain registrar.
```

Comment:
Here it becomes clear that we have overlap, but also scattering of normative information across multiple (sub)sections. I feel this needs restructuring to become more digestible for readers and clearer for implementers. (See high-level comment.)

Reply:
We will take that into account for further restructuring the document.